unit_tests/kernels/UnitTestKernelUtils.h

```diff
@@ -840,6 +860,80 @@ class SSTKernelHex8Mesh : public LowMachKernelHex8Mesh
   sierra::nalu::ScalarFieldType* pecletFactor_{nullptr};
 };
 
+/** Test Fixture for the BLT Gamma Kernels
+ *
+ */
+class BLTGammaM2015KernelHex8Mesh : public LowMachKernelHex8Mesh
```

Comment:
Initially, I actually added some small things to SSTKernelHex8Mesh class, but then I created a new class for the transition model since other turbulence models, like the k-ω or k-ε models, have their own classes. However, as you mentioned, the transition model doesn't necessarily need a separate class because it works with the SST model anyway. Additionally, it only requires minor modifications to SSTKernelHex8Mesh due to the similarity between the transition and turbulence models. I will remove the new class and just add those small things to SSTKernelHex8Mesh.

Reply:
Removed the new class and added some additional lines to SSTKernelHex8Mesh class.